app/models/user.rb

```diff
@@ -37,4 +56,18 @@ def content_count
       universes.length
     ].sum
   end
+
+  private
+
+  # Attributes that are non-public, and should be blacklisted from any public
+  # export (ex. in the JSON api, or SEO meta info about the user)
+  def blacklisted_attributes
```

Comment:
We should probably revisit whitelisting these (or at least verify it's not exposing anything it shouldn't), but this is fine for now. :+1: